backbone/src/main/res/layout/rsb_activity_full_screen_image.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/rsb_black">
+
+    <ImageButton
+        android:id="@+id/fullScreenImageClose"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="16dp"
+        android:layout_marginEnd="16dp"
+        android:background="@null"
+        android:contentDescription="Close"
+        android:src="@drawable/rsb_ic_close"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:ignore="HardcodedText" />
+
+    <ImageView
+        android:id="@+id/fullScreenImage"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="10dp"
+        android:layout_marginEnd="10dp"
+        android:contentDescription="Image"
+        android:scaleType="centerInside"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:ignore="HardcodedText" />
+
+    <TextView
+        android:id="@+id/fullScreenImageEditStep"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="20dp"
+        android:layout_marginEnd="16dp"
+        android:background="?android:attr/selectableItemBackground"
+        android:contentDescription="@string/rsb_edit_step"
+        android:drawableStart="@drawable/rsb_ic_edit"
+        android:drawablePadding="10dp"
+        android:drawableTint="@android:color/white"
+        android:text="@string/rsb_edit_step"
+        android:textAllCaps="true"
+        android:textColor="@android:color/white"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/fullScreenImage"
```

Comment:
What happens if the image (whose bottom is pinned to the bottom of the parent) effectively occupies the full screen? This textView is going to be off-screen and not seen.

Reply:
If I set the image to fill screen, the close and edit buttons may disappear. and if I bind it to the close and edit buttons, the edit button will always be at the bottom of the screen, not right below the image.
What do you suggest we do here?

Reply:
It's not possible to have button that "may disappear" i think.

Reply:
I just fixed that in my latest commit, but I'm still waiting for UI team to get back to me on how to resolve certain cases for this image (i.e. if it gets bigger, or smaller for any reason)

Reply:
According to Felix from UI, The edit button should follow the height of the image. I did this, and made sure if the Image is going to fill the screen, the edit button would still be visible